Doc/library/doctest.rst

```diff
@@ -569,35 +569,20 @@ doctest decides whether actual output matches an example's expected output:
    exception raised is ``ValueError: 3*14``, but will fail, e.g., if
    :exc:`TypeError` is raised.
 
-   It will also ignore the module name used in Python 3 doctest reports. Hence
-   both of these variations will work with the flag specified, regardless of
-   whether the test is run under Python 2.7 or Python 3.2 (or later versions)::
+   It will also ignore the module name used in doctest reports;
```

Comment:
Technically, it ignores not only the module name, but the full qualified name except the name of the exception type (the last component). I would perhaps merge it with the previous paragraph: "... the exception detail (such as the full qualified name or arguments) ..."

It may be worth to explain why this directive is _needed_: the full qualified name and message can be different between different versions of Python or other libraries and between the Python and the C implementations.

Reply:
Thanks; I rewrote the section accordingly to clarify the points you raised and explain why it is needed. I also clarified some potentially confusing/hard to understand phrasing and fixed the code block syntax highlighting, among other minor improvements.